src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -158,7 +158,7 @@ repository:
 
   implicit-expression:
     name: 'meta.expression.implicit.cshtml'
-    begin: '(@)'
+    begin: '(?<![[:alpha:][:alnum:]])(@)'
```

Comment:
How is the first group not captured? I thought you'll need to specify `?:` to make it not capture.

Reply:
Spoke offline. Anything with `(?` is not captured.